enhancements/authentication/direct-external-oidc-provider.md

```diff
@@ -181,16 +208,10 @@ When the built-in OAuth server is used for authentication (the default and origi
 In case an external OIDC provider is configured for authentication, then these controllers and resources are neither useful nor relevant. The operator must watch the OIDC configuration status, and once it becomes available it must deactivate the OAuth stack.
 
 _What signals that the OIDC configuration is available, and when should deactivation start?_
-At the time of writing this enhancement, there is no suitable API that the cluster-kube-apiserver-operator can use to signal that the OIDC configuration is available. The Authentication CR contains an OIDC Clients Status field, however this is specific to clients in the context of the OIDC protocol. Additionally, that field uses the Available/Progressing/Degraded pattern which is not suitable for determining whether the operator needs to deactivate or reactivate the OAuth stack.
 
-The current proposed solution for this is for the authentication operator to inspect the following:
+The cluster-kube-apiserver-operator will update the Authentication CR's `OIDCConfig` status field to signal to the authentication operator that the KAS pods have successfully rolled out with the OIDC configuration in place. 
 
-1. Check all current revisions of the kube-apiserver pods
-2. For each observed revision, check whether it includes a revisioned structured auth-config ConfigMap
-3. If there is, and the respective revision KAS argument is enabled, then OIDC is available on the KAS pods
-4. If any pod is on a revision that does not include an OIDC auth-config, then the OIDC rollout has not been completed in full yet, or that a rollout that switches auth from OIDC back to OAuth is in progress
```

Comment:
We should keep this in to keep documented _how_ the KAS-o will perform the check.